Example/WalletApp/PresentationLayer/Wallet/AuthRequest/AuthRequestView.swift

```diff
@@ -51,8 +33,145 @@ struct AuthRequestView: View {
+                                .foregroundColor(.orange)
+                            
+                            Text("Cannot verify")
+                                .foregroundColor(.orange)
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                
+                        }
+                        .padding(5)
+                        .background(Color.orange.opacity(0.15))
+                        .cornerRadius(10)
+                        .padding(.top, 8)
+                        
+                    case .valid:
+                        HStack(spacing: 5) {
+                            Image(systemName: "checkmark.seal.fill")
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                .foregroundColor(.blue)
+                            
+                            Text("Verified domain")
+                                .foregroundColor(.blue)
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                
+                        }
+                        .padding(5)
+                        .background(Color.blue.opacity(0.15))
+                        .cornerRadius(10)
+                        .padding(.top, 8)
+                        
+                    case .invalid:
+                        HStack(spacing: 5) {
+                            Image(systemName: "exclamationmark.triangle.fill")
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                .foregroundColor(.red)
+                            
+                            Text("Invalid domain")
+                                .foregroundColor(.red)
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                
+                        }
+                        .padding(5)
+                        .background(Color.red.opacity(0.15))
+                        .cornerRadius(10)
+                        .padding(.top, 8)
+                        
+                    case .scam:
+                        HStack(spacing: 5) {
+                            Image(systemName: "exclamationmark.shield.fill")
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                .foregroundColor(.red)
+                            
+                            Text("Security risk")
+                                .foregroundColor(.red)
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                
+                        }
+                        .padding(5)
+                        .background(Color.red.opacity(0.15))
+                        .cornerRadius(10)
+                        .padding(.top, 8)
```

Comment:
nit: Same with these `padding` and `cornerRadius` modifiers. You can wrap the entire switch in a Group and apply them there e.g. Again just to avoid some duplication
```swift
Group { 
 switch foo {
 case .bar:
     HStack { ... }
 }
}
.padding(5)
.cornerRadius(10)
.padding(.top, 8)
```